monitor/main.py

```diff
@@ -88,7 +88,7 @@ def collect(aws):
                         }
                     ]
                 },
-                'Period': CLOUDWATCH_PERIOD * 60,
+                'Period': CLOUDWATCH_PERIOD * 36,
```

Comment:
For future reference:

* CLOUDWATCH_PERIOD * 60 = 5 minutes * 60 = 300 minutes / 60 minutes per hour = 5 hours
* CLOUDWATCH_PERIOD * 36 = 5 minutes * 36 = 180 minutes / 60 minutes per hour = 3 hours